.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
How about 14 instead? I can see someone being on vacation for a week and not responding in time.

Reply:
:+1: I debated about the same thing.